apps/omg_rpc/mix.exs

```diff
@@ -37,11 +37,9 @@ defmodule OMG.RPC.MixProject do
   # Type `mix help deps` for examples and options.
   defp deps do
     [
-      {:phoenix, "~> 1.3.2"},
-      {:cowboy, "~> 1.1"},
+      {:phoenix, "~> 1.3"},
```

Comment:
Wow, `1.3` :question: - smth went wrong

Reply:
no, unless we want to ditch `1.3`. `~> 1.3` includes, among others `1.4.x`. I picked the less stringent way forward.

Do we want to rule `1.3` out the "hard way" :thinking: ?